app/controllers/static_pages_controller.rb

```diff
@@ -16,7 +16,68 @@ def show_category
     filter_posts(published_posts)
   end
 
-  def about; end
+  def about
+    @skills = [
+      {
+        name: 'Java (Android)',
+        percentage: 50,
+        color: '#c0392b'
+      },
+      {
+        name: 'Ruby (Rails)',
+        percentage: 80,
+        color: '#1199ff'
+      },
+      {
+        name: 'PHP (Laravel)',
+        percentage: 20,
+        color: '#9b59b6'
+      },
+      {
+        name: 'Javascript',
+        percentage: 60,
+        color: '#ffa500'
+      },
+      {
+        name: 'HTML/CSS',
+        percentage: 50,
+        color: '#27ae60'
+      }
+    ]
+    @job_timelines = [
+        {
```

Comment:
INFO: Layout/IndentArray: Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.